src/strategies/delegated-ape/index.ts

```diff
@@ -0,0 +1,112 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { getAddress } from '@ethersproject/address';
+import networks from '@snapshot-labs/snapshot.js/src/networks.json';
+import { Multicaller } from '../../utils';
+
+export const author = 'snapshot-labs';
+export const version = '0.1.0';
+
+const abi = [
+  'function delegation(address delegator, bytes32 id) view returns (address delegate)',
+  'function getDelegators(address delegate, bytes32 id) view returns (address[])',
+  'function getEthBalance(address account) public view returns (uint256 balance)'
+];
+
+const CONTRACT_ADDRESS = '0xdd6b74123b2ab93ad701320d3f8d1b92b4fa5202';
```

Comment:
@bonustrack, do you think this should be hardcoded like this or passed in as parameters?